db/models/reviews.js

```diff
@@ -12,6 +12,19 @@ const getAllReviews = async () => {
   }
 };
 
+//get reviews by id
+const getReviewById = async (id) => {
+    try {
+        const { rows: [review] } = await client. query(`
```

Comment:
on line 18,  it looks like there might be a space between 'client.' and 'query' - can you fix this please :)

Reply:
Will do, thanks Lindsay :)